ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.js

```diff
@@ -2930,38 +2931,19 @@ export default {
         return output[data.secondTrineResponse]();
       },
       outputStrings: {
-        north: {
-          en: 'North',
-          de: 'Norden',
-          fr: 'Nord',
-          ja: '北へ',
-          cn: '北',
-          ko: '북쪽으로',
-        },
-        east: {
-          en: 'East',
-          de: 'Osten',
-          fr: 'Est',
-          ja: '東へ',
-          cn: '东',
-          ko: '동쪽으로',
-        },
+        north: Outputs.north,
+        east: Outputs.east,
         south: {
+          // It is a bit different in JA/CN that players should go middle instead of south,
```

Comment:
That is extremely unfortunate that these call different things in different languages.